assets/constants.ts

```diff
@@ -77,6 +77,22 @@ export default {
 
   "RTL_LANGUAGE_CODES": ["ar"],
 
+  // TODO(#14645):Remove these keys once translation service is extended.
```

Comment:
Add space after colon.

It's also unclear where the TODO ends and the regular comment begins. I suggest putting the TODO after the comment, to disambiguate.

Reply:
Done!